client/src/components/App.tsx

```diff
@@ -8,7 +8,8 @@ import Profile from './Profile';
 import StoryBook from './Stories';
 import Ear from './EarStuff/Instrument'
 import Sculpture from './Sculpture'
-import VisualArt from './VisualArt';
+import VisualArt from './Eye/VisualArt';
+import '../styles.css';
```

Comment:
import your styles once in the outermost scope of your client and it should be available within all inner components

Reply:
Yikes, yeah--so the story with that is, once we merged everything together, we noticed some Bulma default styles were overriding some of our CSS styles and causing issues. We read that sometimes it can help to import the CSS stylesheet above the Bulma import in each file, so adding the stylesheet in multiple spots was part of an attempt to do that. Obviously, it didn't work, so we ended up just switching all the Bulma to CSS for time's sake and then forgot to delete all the extra stylesheet imports! (insert single tear emoji) I'll clean em up now!